src/ui/Views/Organisation/Details.cshtml

```diff
@@ -33,12 +42,14 @@
     <div class="govuk-grid-column-two-thirds">
       <div class="course-parts">
         <div class="course-parts__item">
-          <h3 class="course-parts__title">Contact details from UCAS</h3>
-          <dl class="govuk-list--description" role="contentinfo">
-            <dt class="govuk-list--description__label">Training provider code</dt>
-            <dd>@Model.InstitutionCode.ToUpper()</dd>
-            <dt class="govuk-list--description__label">Email</dt>
-            <dd class="@(string.IsNullOrWhiteSpace(Model.EmailAddress) ? "course-parts__fields__value--empty" : "")">
+          <h3 class="course-parts__title">Contact details</h3>
+          <a class="course-parts__action" href="@Url.Action("Contact", "Organisation", new {ucasCode = Model.InstitutionCode})">Edit<span class="govuk-visually-hidden"> your organisation's contact details</span></a>
```

Comment:
I haven't spun this up locally so don't know the context of where this sits in the page, but I think showing the full `Edit your organisation's contact details` is better for all users?

very minor

Reply:
The "Edit" link floats right so i think the full string wouldn't fit :/